io.github.Fontforge.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.Fontforge
+branch: stable
+
+runtime: org.freedesktop.Platform
+runtime-version: "18.08"
+sdk: org.freedesktop.Sdk
+
+command: fontforge
+
+finish-args:
+  - --socket=x11
+  # The file picker does not use portals :(
+  - --filesystem=host
+
+modules:
+  - name: pango
+    sources:
+        - type: archive
+          url: http://ftp.gnome.org/pub/GNOME/sources/pango/1.42/pango-1.42.4.tar.xz
```

Comment:
```suggestion
          url: https://ftp.gnome.org/pub/GNOME/sources/pango/1.42/pango-1.42.4.tar.xz
```

Reply:
I'm not opposing this, but it breaks caching for what I consider a small win.  Do you think the additional authentication via TLS is worth it?

Reply:
How does it break caching?

Reply:
because you cannot have a caching proxy store the content for later use.